warehouse/legacy/api/xmlrpc.py

```diff
@@ -27,14 +27,24 @@
     Role, Project, Release, File, JournalEntry, release_classifiers,
 )
 
-
 pypi_xmlrpc = functools.partial(
     xmlrpc_method,
     endpoint="pypi",
     require_csrf=False,
     require_methods=["POST"],
 )
 
+pypi_xmlrpc_cache_by_project = functools.partial(
+    xmlrpc_method,
```

Comment:
You can partial a partial right? Feels like it'd be slightly nicer not to duplicate this stuff by swapping this from ``xmlrpc_method`` to ``pypi_xmlrpc`` and ditching the shared kwargs.